### ISSUE TITLE
Support more manpage targets

### ISSUE THREAD

**Issue by bingos** [Perl-Toolchain-Gang/ExtUtils-MakeMaker#233]
Following a conversation on #pkgsrc IRC channel on Freenode:

>[11:46] < nbjoerg> BinGOs: from what I see the problem is that installing category 5 man pages is not supported
>[11:46] < nbjoerg> BinGOs: is there a good reason for that?
>[11:47] < BinGOs> well before my time maintaining this beast
>[11:48] < nbjoerg> I can see god reasons for at least category 1, 3, 5, 6, 7 and 8
>[11:48] < nbjoerg> 2, 4 and 9 likely don't make much sense for a perl application

I think we only have 1 and 3 right now.

**Comment by karenetheridge** [comment]
> I think we only have 1 and 3 right now.

Right -- I suspect this would have to change in core first before EUMM can do anything.

**Comment by Leont** [comment]
I'm not sure what would be a good interface for this, but it shouldn't be too hard to implement.

> Right -- I suspect this would have to change in core first before EUMM can do anything.

Not really. Pod::Man supports it fine, it's just that MakeMaker is telling it that it's category 1/3.

**Comment by haarg** [comment]
But the location for the man pages comes from the config vars like installman3dir, which only exist for sections 1 and 3.

**Comment by Leont** [comment]
> But the location for the man pages comes from the config vars like installman3dir, which only exist for sections 1 and 3.

Right, I forgot about that, though in practive it should be easy enough to fake that
